targets/mbedos5/jerryscript-mbed/jerryscript-mbed-drivers/source/DigitalOut.cpp

```diff
@@ -0,0 +1,139 @@
+/* Copyright (c) 2016 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "jerryscript-mbed-util/logging.h"
+#include "jerryscript-mbed-library-registry/wrap_tools.h"
+
+#include "mbed.h"
+
+
+DECLARE_CLASS_FUNCTION(DigitalOut, write) {
+    // check and unbox arguments
+    uintptr_t ptr_val;
+    jerry_get_object_native_handle(this_obj, &ptr_val);
+    DigitalOut* native_ptr = (DigitalOut*)ptr_val;
+
+    jerry_value_t return_value;
+
+    if (args_count == 1 && jerry_value_is_number(args[0])) {
+        int arg0 = int(jerry_get_number_value(args[0]));
+
+        native_ptr->write(arg0);
+        return_value = jerry_create_undefined();
+
+    }
+    else {
+        LOG_PRINT_ALWAYS("invalid arguments in method write");
+        return jerry_create_undefined();
```

Comment:
It might be better perhaps to return an error object with an error message (e.g., with `jerry_create_error(JERRY_ERROR_TYPE, "...")`), instead of logging and returning undefined.